packages/diff-sequences/__benchmarks__/test.js

```diff
@@ -5,19 +5,23 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-// Make sure to run node with --expose-gc option!
```

Comment:
This comment was also important. It is replaced with `throw new Error()` and "How to start the test" instruction (;